pkgs/data/fonts/iosevka/default.nix

```diff
@@ -48,23 +48,28 @@
 , extraParameters ? null
   # Custom font set name. Required if any custom settings above.
 , set ? null
+, pre28 ? false
 }:
 
 assert (privateBuildPlan != null) -> set != null;
 assert (extraParameters != null) -> set != null;
 
 buildNpmPackage rec {
-  pname = if set != null then "iosevka-${set}" else "iosevka";
-  version = "27.3.5";
+  pname =
+    if !pre28
+    then if set != null then "Iosevka${set}" else "Iosevka"
+    else if set != null then "iosevka-${set}" else "iosevka";
+
```

Comment:
Perhaps this pre28 stuff can just be moved into the comfy package? As `buildNpmPackage` doesn't support function based derivations (i.e. `stdenv.mkDerivation (finalAttrs: ...)`) we can't really have an elegant solution here.